src/app/views/media-coverage/media-coverage.component.ts

```diff
@@ -8,4 +8,26 @@ import { media_coverage } from '../../../data-entries/json/media-coverage';
 })
 export class MediaCoverageComponent {
   media_coverage = media_coverage;
+
+  @ViewChild('scrollToTopButton') scrollToTopButton!: ElementRef;
```

Comment:
Remove this and related

Reply:
May have to implement in other way altogether.